.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
+  "recommendations": [
+    "ruby-syntax-tree.vscode-syntax-tree",
+    "shopify.ruby-lsp"
+  ]
+}
```

Comment:
Not a VSC user What does that do? 
Is it necessary to have that on the repo? 

If there is no impact to non VSC user this should be ok ;)

Reply:
No impact for non-vsc users indeed (the folder will just be ignored).

This file recommends (but does not force you) to install 2 extensions in VSCode (that are also recommended in the monolith's `.vscode/extension.json` file). These extensions provide you with the necessary tooling to work in a Ruby codebase (class definitions/sources on hover, integration with the debugguer, ...).